src/core/room/room.server.ts

```diff
@@ -647,7 +647,15 @@ export class ServerRoom extends Room<WorkPlace.Server> {
       return event.responsedEvent;
     }
 
-    let responseEvent: ClientEventFinder<GameEventIdentifiers.AskForCardUseEvent> | undefined;
+    let responseEvent: ClientEventFinder<GameEventIdentifiers.AskForCardUseEvent> | undefined = {
+      fromId: to,
+    };
+
+    if (this.isGameOver()) {
```

Comment:
need a systematic solution to resolve all this kind of issues in different triggers